src/test/java/org/jvnet/hudson/test/RealJenkinsRuleTest.java

```diff
@@ -179,6 +189,42 @@ private static void _stepsDoNotRunOnHttpWorkerThread(JenkinsRule r) throws Throw
         assertNull(Stapler.getCurrentRequest());
     }
 
+
+    @Test
+    public void test500Errors() throws IOException {
```

Comment:
an IT with a plugin that throws an exception in an initializer that would block jenkins startup would be good here, as that is what you are making these changes for?